examples/quickstart/hello_world.py

```diff
@@ -0,0 +1,156 @@
+from pipecat.pipeline.task import PipelineParams, PipelineTask
+from pipecat.processors.aggregators.openai_llm_context import OpenAILLMContext
+from pipecat.serializers.twilio import TwilioFrameSerializer
+from pipecat.services.cartesia.stt import CartesiaSTTService
+from pipecat.services.cartesia.tts import CartesiaTTSService
+from pipecat.services.gemini_multimodal_live.gemini import GeminiMultimodalLiveLLMService
+from pipecat.services.google.llm import GoogleLLMService
+from pipecat.services.openai.llm import OpenAILLMService
+from pipecat.transports.base_transport import BaseTransport, TransportParams
+from pipecat.transports.network.fastapi_websocket import (
+    FastAPIWebsocketParams,
+    FastAPIWebsocketTransport,
+)
+from pipecat.transports.network.small_webrtc import SmallWebRTCTransport
+from pipecat.transports.network.webrtc_connection import IceServer, SmallWebRTCConnection
+from pipecat.transports.services.daily import DailyParams, DailyTransport
+from pipecat.utils.text.markdown_text_filter import MarkdownTextFilter
+
+from pipecat_flows import (
+    FlowArgs,
+    FlowConfig,
+    FlowManager,
+    FlowResult,
+    FlowsFunctionSchema,
+    NodeConfig,
+)
+
+load_dotenv(override=True)
+
+transport_params = {
+    "daily": lambda: DailyParams(
+        audio_in_enabled=True, audio_out_enabled=True, vad_analyzer=SileroVADAnalyzer()
+    ),
+    "twilio": lambda: FastAPIWebsocketParams(
+        audio_in_enabled=True, audio_out_enabled=True, vad_analyzer=SileroVADAnalyzer()
+    ),
+    "webrtc": lambda: TransportParams(
+        audio_in_enabled=True, audio_out_enabled=True, vad_analyzer=SileroVADAnalyzer()
+    ),
+}
+
+# Flow nodes
+def create_initial_node() -> NodeConfig:
+    """Start here."""
+    get_favorite_color_func = FlowsFunctionSchema(
+                name="get_favorite_color",
+                description="Record the color the user said is their favorite.",
+                required=["color"],
+                handler=print_favorite_color_and_set_next_node,
+                properties={"color": {"type": "string"}},
+            )
+
+    return {
+        "name": "initial",
+        "role_messages": [
+            {
+                "role": "system",
+                "content": "You are an inquisitive child. Use very simple language. Ask simple questions. You must ALWAYS use one of the available functions to progress the conversation. Your responses will be converted to audio. Avoid outputting special characters and emojis.",
+            }
+        ],
+        "task_messages": [
+            {"role": "system", "content": "Say 'Hello world' and ask what is the user's favorite color."}
+        ],
+        "functions": [get_favorite_color_func],
+    }
+
+async def print_favorite_color_and_set_next_node(args: FlowArgs, flow_manager: FlowManager) -> tuple[str, NodeConfig]:
+    print(f"Your favorite color is: {args["color"]}")
+    return args["color"], end_conversation()
+
+def end_conversation() -> NodeConfig:
```

Comment:
I'd recommend calling this `create_end_node()`, in keeping with the `create_*_node()` naming convention.